docs/UserGuide.md

```diff
@@ -88,13 +78,14 @@ There are 5 CLI command formats as shown below:
 1) `COMMAND_WORD`
 2) `COMMAND_WORD INDEX`
 3) `COMMAND_WORD PARAMETER`
-3) `COMMAND_WORD PARAMETERS`
-4) `COMMAND_WORD INDEX PARAMETERS`
-</box>
+4) `COMMAND_WORD PARAMETERS`
```

Comment:
actually we dont have any commands that follow the COMMAND_WORD PARAMETERS format right?
should we remove point 4 entirely?

Reply:
I think `find` and `scheduledate` command has this form!

Reply:
I think `find` and `scheduledate` command has this form!